pom.xml

```diff
@@ -78,12 +79,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>scm-api</artifactId>
-            <version>2.1.0</version>
+            <version>2.6.3</version>
```

Comment:
Relying on the BOM (see doc https://jenkins.io/doc/developer/plugin-development/dependency-management/) would avoid manually setting up versions for the plugin dependencies.